core/domain/html_validation_service.py

```diff
@@ -1116,6 +1116,11 @@ def add_math_content_to_math_rte_components(html_string):
         html_string.encode(encoding='utf-8'), 'html.parser')
     for math_tag in soup.findAll(name='oppia-noninteractive-math'):
         if math_tag.has_attr('raw_latex-with-value'):
+            # There have been cases where the attr value is empty.
+            if not math_tag['raw_latex-with-value']:
+                math_tag.decompose()
```

Comment:
Just for clarity: what does decompose do?

Reply:
It removes the tag.

Reply:
Thanks! I inferred that from the tests, but wanted to make sure.